eng/versioning/pom_file_version_scanner.ps1

```diff
@@ -527,7 +527,7 @@ Get-ChildItem -Path $Path -Filter pom*.xml -Recurse -File | ForEach-Object {
         {
             # unfortunately because there are POM exceptions we need to wildcard the group which may be
             # something like <area>_groupId
-            if ($versionNode.NextSibling.Value.Trim() -notmatch "{x-version-update;(\w+)?$($groupId):$($artifactId);\w+}")
+            if ($versionNode.NextSibling.Value.Trim() -notmatch "{x-version-update;(.+)?$($groupId):$($artifactId);\w+}")
```

Comment:
Can this regex be declared as a constant instead of updating it in multiple places?
 `(.+)?$($groupId):$($artifactId)`